kubernetes/base/config/modules.yml

```diff
@@ -0,0 +1,70 @@
+##
+## Modules enabled in all ejabberd virtual hosts.
+##
+modules:
+  mod_blocking: {} # requires mod_privacy
+  mod_disco: {}
+  mod_pres_counter:
+    count: 100
+    interval: 60
+  mod_privacy: {}
+  mod_vcard:
+    search: true
+  ModBeamStats: {}
+  mod_pottymouth:
+    blacklists:
+      default: /home/ejabberd/.ejabberd-modules/sources/mod_pottymouth/blacklist/blacklist_en.txt
+      en: /home/ejabberd/.ejabberd-modules/sources/mod_pottymouth/blacklist/blacklist_en.txt
+  ModPushSkillz:
+    rabbit_url: "amqp://skillz-user:REDACTED@rabbit.qa.skillz.com:5672/skillz"
```

Comment:
I think ejabberd fails to start if it can't reach the rabbit server for mod push skillz. Can this module be disabled just in base ?

Reply:
Base is basically considered dev, so we should only be using that for local development. 

Going to take you up on this being optional, since getting this to work with our local development stack is a much larger issue.